project/plugins.sbt

```diff
@@ -1,10 +1,13 @@
 
 resolvers += Resolver.sonatypeRepo("public")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.1")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15")
+
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
```

Comment:
This change seem orthogonal, I would suggest putting it into another PR (if it's really necessary - I generally tend to avoid changing dependencies unless it's absolutely needed).